CMakeLists.txt

```diff
@@ -197,7 +197,7 @@ set(LIBSOUNDIO_LIBS
 set(CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS_DEBUG} -Werror -pedantic")
 
 
-set(LIB_CFLAGS "-std=c11 -fvisibility=hidden -Wall -Werror=strict-prototypes -Werror=old-style-definition -Werror=missing-prototypes -D_REENTRANT -D_POSIX_C_SOURCE=200809L")
+set(LIB_CFLAGS "-std=c11 -fvisibility=hidden -Wall -Werror=strict-prototypes -Werror=old-style-definition -D_REENTRANT -D_POSIX_C_SOURCE=200809L")
```

Comment:
If we need to remove this warning, instead of removing it completely, we can move `-Werror=missing-prototypes` to the line up there with the other debug-only flags (`-Werror -pedantic`).